src/pocketmine/Player.php

```diff
@@ -3590,6 +3591,25 @@ public function sendTranslation($message, array $parameters = []){
 		$this->dataPacket($pk);
 	}
 
+	public function sendWeather(){
```

Comment:
This is core Level functionality, not Player. It should be broadcasted in the same way that time is.